portfolio/src/main/webapp/js/script.js

```diff
@@ -49,13 +49,25 @@ function makeChangeDescriptionCallBack(projectIndex) {
     }
 }
 
+/**
+ * Initialize map.
+ */
+var map;
+
+function initMap() {
+  map = new google.maps.Map(document.getElementById("map"), {
+    center: { lat: 59.944186, lng: 30.306510 },
+    zoom: 13
```

Comment:
Add a comment to explain where those numbers come from

Reply:
Alternatively, you can create a variable (e.g. `stPetersburgCentre`) and use it here, e.g.  `{ lat: stPetersburgCentre.latitude, ... }`